apps/laboratory/tests/siwe-email.spec.ts

```diff
@@ -18,6 +18,7 @@ const emailSiweTest = test.extend<{ library: string }>({
 emailSiweTest.describe.configure({ mode: 'serial' })
 
 emailSiweTest.beforeAll(async ({ browser, library }, testInfo) => {
+  emailSiweTest.setTimeout(120000)
```

Comment:
is this not needed on email.spec as well?

Reply:
It's there